keps/sig-release/20190316-rebase-images-to-distroless.md

```diff
@@ -0,0 +1,167 @@
+3. Images based "FROM scratch" is safe to switch to distroless/static directly.
+4. Images from kubernetes incubator won't be changed directly by this KEP and the release plan is not estimated here. We notify the project OWNERs and we defer to the OWNERs on whether or not those images should be updated.
+
+### For Core Master Images
+
+The core master images includes `kube-apiserver`, `kube-controller-manager`, `kube-scheduler` and `kube-proxy`. For `kube-proxy`, it is based on debian-iptable which distroless/static doesn't support iptables yet. Thus, kube-proxy won't be changed.
+
+Currently, there are **three** different workflows to build the core master images and which workflow to use  is determined by each **cloud provider**.
+
+####[Bash Release](https://github.com/kubernetes/kubernetes/blob/caf9d94d697ce327e0c1c3dee71a1f06a6fc918e/build/root/Makefile#L419)
+
+Run `make quick-release` under kubernetes repo. This approach is most commonly used and it uses the bash scripts (See [build/release.sh](https://github.com/kubernetes/kubernetes/blob/caf9d94d697ce327e0c1c3dee71a1f06a6fc918e/build/release.sh#L36) for details) to build the images. In this workflow, the base image is specified in the [`build/common.sh`](https://github.com/kubernetes/kubernetes/blob/f26048ecb1c7b6fb67c2e7c7c96070d7a1743d86/build/common.sh#L96).
+
+####[Bazel Release](https://github.com/kubernetes/kubernetes/blob/caf9d94d697ce327e0c1c3dee71a1f06a6fc918e/build/root/Makefile#L604)
+
+Run `make bazel-release` under kubernetes repo. This approach uses bazel to build the image artifact based on this [BUILD](https://github.com/kubernetes/kubernetes/blob/caf9d94d697ce327e0c1c3dee71a1f06a6fc918e/build/release-tars/BUILD) rule (More details in [bazel.bzl/release-filegroup](https://github.com/kubernetes/repo-infra/blob/4528e18f5d62a2a5172f76d198738d85d4d04734/defs/build.bzl#L134)) . In this workflow, the base image is specified in the [build/BUILD](https://github.com/kubernetes/kubernetes/blob/3fd6f97f55d51c01df3c01c7ffbb2834c25d9900/build/BUILD#L31) rule.
+
+####[Test Release](https://github.com/kubernetes/test-infra/tree/master/kubetest)
+
+Run `kubetest` or `hack/e2e`. See details in the [test-infra repo](https://github.com/kubernetes/test-infra/tree/master/kubetest). This approach is recommended for development testing and is broadly used by contributors. However, this approach is under a test env and it uses different config than the two official workflows as described above. In this workflow, the base image is specified to use k8s.gcr.io/pause:3.1.
+
+###Solution
```

Comment:
need a space after `###` for markdown to render correctly

Reply:
I see. 
I was using the sublime markdown build, it seems using a different render.

Reply:
Yeah, there's some variation in how they tolerate things like this. I recommend looking at it on your github fork after you push it from your local machine.